medusa/databases/main_db.py

```diff
@@ -850,6 +844,14 @@ def shift_tv_qualities(self):
         for result in sql_results:
             quality = result[b'quality']
             new_quality = quality << 1
+
+            # UNKNOWN quality value is 65536 (1 << 16) instead of 32768 (1 << 15) after the shift
+            # Qualities in the tv_shows table have the combined values of allowed and preferred qualities.
+            # Preferred quality couldn't contain UNKNOWN
+            if new_quality & 65536 > 0:  # If contains UNKNOWN allowed quality
+                new_quality -= 65536  # Remove it
+                new_quality |= common.Quality.UNKNOWN  # Then re-add it using the correct value
```

Comment:
Maybe this should be `new_quality |= 1`, in case we're changing the value again.

Reply:
I think it's good as it is now.